.travis.yml

```diff
@@ -1,10 +1,10 @@
 # This Travis job script has been generated by a script via
 #
-#   haskell-ci '--branches' 'master' '--jobs-selection=any' '--output' '.travis.yml' 'cabal.project'
+#   haskell-ci '--config' 'cabal.haskell-ci' 'cabal.project'
 #
 # For more information, see https://github.com/haskell-CI/haskell-ci
 #
-# version: 0.3.20190327
+# version: 0.3.20190730
```

Comment:
Wait, what version of `head.hackage` is this? :)

Reply:
The one which doesn't treat 8.8 as needing head.hackage, yet it's made configurable in https://github.com/haskell-CI/haskell-ci/pull/289